File: athena/src/main/java/br/com/athena/model/Atividade.java
package br.com.athena.model;

import javax.persistence.*;

@Entity
public class Atividade {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    private String titulo;

    private String descricao;

    @ManyToOne
    private Aula aula;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }
}

File: athena/src/app/model/Atividade.ts
import {Aula} from "./Aula";

export interface Atividade {
  "id": number,
  "titulo": number,
  "descricao": string
  "link": string[],
  "aula": Aula,
  "data_de_publicacao": Date,
  "data_de_entrega": Date
}
